Exercises/numbers/numbers.js

```diff
@@ -0,0 +1,53 @@
+//1. Assign by hand same value to 3 variables in binary, octal and hexadecimal format.
+//const decimal = new Number('99');
+
+const binary = new Number('1100011');
+const octal = new Number('143');
+const hexadecimal = new Number(63);
```

Comment:
to do this task, you need just to convert somewhere some decimal number in other calculation system, and assign to variable in a string format. The idea is that you can't say js interpreter to treat number as octal or hex, you can do it only with string, and then parse string in needed calculation system.

Reply:
thanks! updated as const binary = '0000000001111111';